api/src/main/java/org/pmiops/workbench/profile/ProfileMapper.java

```diff
@@ -39,6 +43,7 @@ public interface ProfileMapper {
   @Mapping(source = "dbUser.duccAgreement.userInitials", target = "duccSignedInitials")
   @Mapping(source = "dbUser.duccAgreement.completionTime", target = "duccCompletionTimeEpochMillis")
   @Mapping(source = "dbUser.demographicSurveyV2", target = "demographicSurveyV2")
+  @Mapping(target = "newUserSatisfactionSurveyEligibility", ignore = true)
```

Comment:
if you pass in the boolean value instead (see comment in ProfileService) and name the param to match the API entity field, MapStruct will figure out the mapping automatically and this line can be removed.